spruned/daemon/electrod/electrum_servers.json

```diff
@@ -125,6 +125,9 @@
     ["y4td57fxytoo5ki7.onion", "s"]
   ],
   "bc_testnet": [
+    ["tn.not.fyi", 5502],
```

Comment:
There's a shortage of public Electrum testnet servers. 
I remember I've removed those servers from the list because spruned wasn't able to pass the first check on connect (fetch a raw transaction) from them. I'm afraid they don't have the rawtxindex. As far as I know, it is not mandatory to keep a server up & running for the average Electrum client, but spruned needs it.

As far as I know, by the way, the "personal electrum server" db-less interface wraps enough API to serve spruned' requests, if the raw transaction index is enabled. It's easier to ask people with full nodes to run a small "personal electrum server" in front of their full node, instead of a resources-consuming electrumx.

I'll open an issue to investigate the topic. 
A test script for servers would be helpful at that point.

Reply:
Ah, it's ok to merge anyway, until the issue is proper addressed! Maybe the owners updated their servers.